### PR TITLE
Release 0.14.0 alpha2

### PR DESCRIPTION
**Pull request by ssd71**

### CODE REVIEW

File: packages/graphback-codegen-offix/README.md [hunk 1]
```diff
@@ -14,4 +14,4 @@
 
 Offix Code Generator. 
 
-> NOTE: This package is an early alpha and not officially supported by Graphback
+### NOTE: This package is now *deprecated* and will not be receiving any further updates. Please see [this](https://github.com/aerogear/graphback/issues/1358) for details.
```
Comment (machi1990) [inline packages/graphback-codegen-offix/README.md:17]: Seeing the issue, it talks about renaming the package too. Should we do it now? @wtrocki

File: packages/graphback-cli/package.json [hunk 2]
```diff
@@ -49,10 +49,10 @@
     "execa": "4.0.2",
     "figlet": "1.4.0",
     "glob": "7.1.6",
-    "graphback": "0.14.0-alpha1",
+    "graphback": "^0.14.0-alpha2",
```
Comment (machi1990) [inline packages/graphback-cli/package.json:52]:
```suggestion
    "graphback": "0.14.0-alpha2",
```

File: packages/graphback-cli/package.json [hunk 3]
```diff
@@ -49,10 +49,10 @@
...
     "graphql": "^14.6.0",
     "graphql-config": "3.0.2",
-    "graphql-migrations": "0.14.0-alpha1",
+    "graphql-migrations": "^0.14.0-alpha2",
```
Comment (machi1990) [inline packages/graphback-cli/package.json:55]:
```suggestion
    "graphql-migrations": "0.14.0-alpha2",
```

File: packages/graphback-codegen-client/package.json [hunk 4]
```diff
@@ -34,7 +34,7 @@
     "typescript": "3.9.3"
   },
   "dependencies": {
-    "@graphback/core": "0.14.0-alpha1",
+    "@graphback/core": "^0.14.0-alpha2",
```
Comment (machi1990) [inline packages/graphback-codegen-client/package.json:37]:
```suggestion
    "@graphback/core": "0.14.0-alpha2",
```

**Comment by wtrocki** [comment]
I always forget to mention that we use pinned versions. Thx @machi1990 for review

**Comment by machi1990** [comment]
Awesome work @ssd71 you missed a couple other version pins, can you do a search + replace? `/s/^0.14.0-alpha2/0.14.0-alpha2`

File: packages/graphback/package.json [hunk 5]
```diff
@@ -32,10 +32,10 @@
...
   "license": "Apache-2.0",
   "dependencies": {
-    "@graphback/codegen-client": "0.14.0-alpha1",
-    "@graphback/codegen-schema": "0.14.0-alpha1",
-    "@graphback/core": "0.14.0-alpha1",
-    "@graphback/runtime": "0.14.0-alpha1",
+    "@graphback/codegen-client": "^0.14.0-alpha2",
+    "@graphback/codegen-schema": "^0.14.0-alpha2",
+    "@graphback/core": "^0.14.0-alpha2",
+    "@graphback/runtime": "^0.14.0-alpha2",
```
Comment (machi1990) [inline packages/graphback/package.json:38]: should be `0.14.0-alpha2`

File: templates/ts-apollo-mongodb-backend/package.json [hunk 6]
```diff
@@ -12,12 +12,12 @@
   },
   "license": "Apache 2.0",
   "dependencies": {
-    "@graphback/runtime-mongo": "0.14.0-alpha1",
+    "@graphback/runtime-mongo": "^0.14.0-alpha2",
```
Comment (machi1990) [inline templates/ts-apollo-mongodb-backend/package.json:15]: should be `0.14.0-alpha2`

File: templates/ts-apollo-mongodb-backend/package.json [hunk 7]
```diff
@@ -12,12 +12,12 @@
...
     "apollo-server-express": "2.14.1",
     "cors": "2.8.5",
     "dotenv": "8.2.0",
     "express": "4.17.1",
-    "graphback": "0.14.0-alpha1",
+    "graphback": "^0.14.0-alpha2",
```
Comment (machi1990) [inline templates/ts-apollo-mongodb-backend/package.json:20]: should be `0.14.0-aplha2`

File: templates/ts-apollo-postgres-backend/package.json [hunk 8]
```diff
@@ -12,15 +12,15 @@
   },
   "license": "Apache 2.0",
   "dependencies": {
-    "@graphback/runtime-knex": "0.14.0-alpha1",
+    "@graphback/runtime-knex": "^0.14.0-alpha2",
```
Comment (machi1990) [inline templates/ts-apollo-postgres-backend/package.json:15]: should be `0.14.0-alpha2`

File: templates/ts-apollo-postgres-backend/package.json [hunk 9]
```diff
@@ -12,15 +12,15 @@
...
     "apollo-server-express": "2.14.1",
     "cors": "2.8.5",
     "dotenv": "8.2.0",
     "express": "4.17.1",
-    "graphback": "0.14.0-alpha1",
+    "graphback": "^0.14.0-alpha2",
```
Comment (machi1990) [inline templates/ts-apollo-postgres-backend/package.json:20]: same here

File: templates/ts-apollo-postgres-backend/package.json [hunk 10]
```diff
@@ -12,15 +12,15 @@
...
     "graphql": "14.6.0",
     "graphql-config": "3.0.2",
-    "graphql-migrations": "0.14.0-alpha1",
+    "graphql-migrations": "^0.14.0-alpha2",
```
Comment (machi1990) [inline templates/ts-apollo-postgres-backend/package.json:23]: same here, let's pin the version

File: templates/ts-react-apollo-client/package.json [hunk 11]
```diff
@@ -1,14 +1,14 @@
...
+  "version": "0.14.0-alpha2",
   "private": true,
   "devDependencies": {
     "@graphql-codegen/add": "1.15.0",
     "@graphql-codegen/cli": "1.15.0",
     "@graphql-codegen/typescript": "1.15.0",
     "@graphql-codegen/typescript-operations": "1.15.0",
     "@graphql-codegen/typescript-react-apollo": "1.15.0",
-    "graphback-cli": "0.14.0-alpha1",
+    "graphback-cli": "^0.14.0-alpha2",
```
Comment (machi1990) [inline templates/ts-react-apollo-client/package.json:11]: let's pin the version.

**Comment by ssd71** [comment]
Sorry @machi1990 ! Thanks for pointing out!

**Comment by machi1990** [comment]
> Sorry @machi1990 ! Thanks for pointing out!

Nice work. Thanks for taking care of the comments :-)

**Pull request merged**